src/pytype.jl

```diff
@@ -363,13 +363,46 @@ function pyjlwrap_hash32(o::PyPtr)
     return h == reinterpret(UInt32, Int32(-1)) ? pysalt32 : h::UInt32
 end
 
+# just string(Docs.doc(f)) doesn't work on VERSION < v"0.5"
+function docstring(f::Function)
```

Comment:
Should not be restricted to `::Function`.

Now that we require 0.5, this can presumably become
```jl
docstring(x) = string(Docs.doc(x))
```